auto_submit/test/src/service/fake_github_service.dart

```diff
@@ -276,4 +284,31 @@ class FakeGithubService implements GithubService {
   Future<Issue> getIssue({required RepositorySlug slug, required int issueNumber}) async {
     return githubIssueMock!;
   }
+
+  /// If useMergeRequestMockList is true then we will return elements from that
+  /// list until it is empty. The developer should track the number of times this
+  /// method is called as managing an empty list is not done here.
```

Comment:
Dart doc format is a single sentence, followed by a blank link, with a longer description afterwards.

https://dart.dev/guides/language/effective-dart/documentation#do-start-doc-comments-with-a-single-sentence-summary